daemon/cmd/daemon_main.go

```diff
@@ -506,6 +506,9 @@ func init() {
 	flags.MarkHidden(option.KVstoreLeaseTTL)
 	option.BindEnv(option.KVstoreLeaseTTL)
 
+	flags.Int(option.MaxConsecutiveQuorumErrorsName, defaults.MaxConsecutiveQuorumErrors, "Max acceptable consecutive quorum errors before the agent assumes permanent failure")
```

Comment:
Might be worth to mention that this is related to etcd in the usage text, e.g.

```suggestion
	flags.Int(option.MaxConsecutiveQuorumErrorsName, defaults.MaxConsecutiveQuorumErrors, "Max acceptable consecutive etcd quorum errors before the agent assumes permanent failure")
```

Reply:
Or perhaps it should say k/v store?